yt_dlp/extractor/bilibili.py

```diff
@@ -482,24 +528,24 @@ def _real_extract(self, url):
         bangumi_id = self._match_id(url)
 
         # Sometimes this API returns a JSONP response
-        season_info = self._download_json(
-            'http://bangumi.bilibili.com/jsonp/seasoninfo/%s.ver' % bangumi_id,
-            bangumi_id, transform_source=strip_jsonp)['result']
+        webpage = self._download_webpage(url, bangumi_id)
+
+        json__INITIAL_STATE__ = self._parse_json(
+            self._search_regex(r'window.__INITIAL_STATE__\s*=\s*({.+?});\(function\(\)', webpage,
+                               '__INITIAL_STATE__', default=None) or '{}', bangumi_id, fatal=False)
+
+        bangumi_id = traverse_obj(json__INITIAL_STATE__, ('epInfo', 'aid'))
+        season_info = json__INITIAL_STATE__['mediaInfo']
 
         entries = [{
             '_type': 'url_transparent',
-            'url': smuggle_url(episode['webplay_url'], {'no_bangumi_tip': 1}),
+            'url': episode['link'],
             'ie_key': BiliBiliIE.ie_key(),
-            'timestamp': parse_iso8601(episode.get('update_time'), delimiter=' '),
-            'episode': episode.get('index_title'),
-            'episode_number': int_or_none(episode.get('index')),
+            'timestamp': None,
```

Comment:
why?

Reply:
> why?

There is no episode_number and timestamp in this html (url)
And I think there is no need for request some seasoninfo api only for it

Reply:
Don't set timestamp to None. Just remove the key

Reply:
ok